AjobK/Athena
# 259 homepage infinite scrolling

closes #259 

<br />

 **What is it supposed to do**  
When the user has fetched all posts on the homepage, scrolling further would repeat the same process to allow infinite scrolling.

<br />

 **How can we test your branch**  
Go to the homepage and scroll down. See if all posts that should be fetched are properly fetched and displayed again.

<br />

**1. General checklist**

<br />

**2. If relevant, front-end checklist**

<br />

**3. If relevant, back-end checklist**

<br />

**4. If relevant, test these browsers**
